File: PodrziMe.Model/Korisnik.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PodrziMe.Model
{
    public class Korisnik
    {
        public int KorisnikId { get; set; }

        public string? Email { get; set; }

        public string? Telefon { get; set; }

        public string KorisnickoIme { get; set; } = null!;

        public bool? Status { get; set; }

        public int? UlogaId { get; set; }

        public virtual Uloga? Uloga { get; set; }

    }
}

File: UI/podrzime_admin/lib/models/korisnik.dart
import 'package:json_annotation/json_annotation.dart';

part 'korisnik.g.dart';

@JsonSerializable()
class Korisnik {
  int? korisnikId;
  String? email;
  String? telefon;
  String? korisnickoIme;
  String? lozinka;
  String? lozinkaPotvrda;
  bool? status;
  int? ulogaId;

  Korisnik({
    this.korisnikId,
    this.email,
    this.telefon,
    this.korisnickoIme,
    this.lozinka,
    this.lozinkaPotvrda,
    this.status,
    this.ulogaId,
  });

  factory Korisnik.fromJson(Map<String, dynamic> json) =>
      _$KorisnikFromJson(json);

  Map<String, dynamic> toJson() => _$KorisnikToJson(this);
}
